src/modules/shared.mjs

```diff
@@ -105,92 +76,227 @@ export function addFeedbackLink() {
   document.body.appendChild(aside);
 }
 
-export function updateLogo() {
-  const logo = document.getElementsByTagName("img")[0];
-  logo.src = "https://beta.nj.gov/files/dol_logo.png";
-  logo.style.removeProperty("width");
-  logo.style.removeProperty("height");
-  logo.style.marginLeft = isDesktop() ? "107px" : "18px";
-  logo.alt = "Official logo for the New Jersey Department of Labor";
-  logo.parentElement.parentElement.removeAttribute("height");
+// Used for tagged template strings
+export function html(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
 }
 
-// Used for tagged template strings
-const html = (strings, ...values) => String.raw({ raw: strings }, ...values);
+export function js(strings, ...values) {
```

Comment:
Dust: I could just be bad at searching but is this used anywhere?

Reply:
You are correct, I think I had it and then removed it